test/distributed/org/apache/cassandra/distributed/test/PaxosRepairTest.java

```diff
@@ -204,10 +204,10 @@ private static void repair(Cluster cluster, String keyspace, String table)
     private static final Consumer<IInstanceConfig> CONFIG_CONSUMER = cfg -> {
```

Comment:
I know your changes don't have anything to do with it, but there are tons of unused imports in `PaxosRepairTest`

Reply:
Removed, but nothing else.... it has 29 warnings and I am the last person to touch around Paxos at the moment :-)